src/hooks/useAsyncEffect.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+export default function useAsyncEffect(
```

Comment:
is this basically just to bypass the eslint rule? Can't we just ignore this in a place where we're using it?

Reply:
No, not really. You can't use the `async function` as a callback of `useEffect`.
Another patter you can use is:

```js
useEffect(() => {
  (async () => {
  
  })()
}, []);
```

And this is way worse.